http/api/transport.go

```diff
@@ -115,6 +116,7 @@ func decodeRequest(ctx context.Context, r *http.Request) (interface{}, error) {
 		Channel:     chanID,
 		Subtopic:    subtopic,
 		Payload:     payload,
+		Created:     time.Now().Unix(),
```

Comment:
This value will be in whole seconds. And I think that I had problem when I sent more message with same time that they were overwritten in time series databases.
Maybe consider to make this float here: `float64(time.Now().UnixNano()) / float64(1e9)` and in proto file also declare is as `double` to make this value more precise.

Reply:
I agree on this!

I think we even set up InfluxDB by default on microsecond precision...

I would suggest using Unix time (integer), but as precise as possible (`UnixNano()` for example)